modules/geolocation/geolocation.py

```diff
@@ -0,0 +1,99 @@
+"""
+Geolocation module to map pixel coordinates to geographical coordinates
+"""
+
+import numpy as np
+
+class Geolocation:
+    """
+    Locates the geographical position of a set of pixels
+    """
+
+    # TODO Class members
+    def __init__(self):
+        """
+        Initial setup of class members
+
+        Returns
+        -------
+        Geolocation
+        """
+
+        # Input to gather_point_pairs()
+        self.__cameraOrigin3o = np.array([0.0, 0.0, 2.0])
+        self.__cameraDirection3c = np.array([0.0, 0.0, -1.0])
+        self.__cameraOrientation3u = np.array([1.0, 0.0, 0.0])
+        self.__cameraOrientation3v = 1 * np.cross(self.__cameraDirection3c, self.__cameraOrientation3u)
+        self.__cameraResolution = np.array([1000, 1000])  # TODO Make global?
```

Comment:
Is this the actual value? If not, we need a TODO here.

Reply:
This is also a placeholder.

Reply:
Got it